### PR TITLE
Add resolution repo to org.yaml

### PR DESCRIPTION
**Pull request by sbwsg**
The resolution repo has just been spun up to host everything related to remote resolution in tekton.

This PR initializes the org.yaml entries for the resolution project, adding two teams: maintainers and collaborators.

Fixes https://github.com/tektoncd/community/issues/611

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: org/org.yaml
Comment (afrittoli) [inline org/org.yaml:731]: `maintainers` is the maintainers of the GitHub team. I think using `members` would be the appropriate field to use.
Unfortunately peribolos (or GitHub?) does not allow for an empty `maintainers` list - which would be ideal because we want the teams to be maintained through periobolos only. For other teams I used members of the governance team as maintainers of teams, but that too is not ideal, because members of the GB change over time.
Perhaps we could use a bot account as `maintainers`?

**Comment by sbwsg** [comment]
/hold

Adding hold while i correct the org yaml just to make sure it doesn't merge in a broken state.

**Comment by afrittoli** [comment]
We have an org file linter CI job, I'm not sure why it's not running...

**Comment by afrittoli** [comment]
Oh, it is running, but something is wrong with one of the checks https://dashboard.dogfooding.tekton.dev/#/namespaces/tekton-ci/pipelineruns/pull-community-org-validation-8dmbw?pipelineTask=check-git-files-changed&step=check-files-changed

**Comment by afrittoli** [comment]
It was broken here - https://github.com/tektoncd/plumbing/pull/975/files
Because the environment variable is used now, single quotes there do not work anymore as they prevent the expansion of the env variable.

File: org/org.yaml
Comment (sbwsg) [reply org/org.yaml]: I've added the governing board members as maintainers on the teams for now until a bot account is set up.

**Comment by sbwsg** [comment]
I've removed the `resolution.collaborators` section of the org.yaml to see if it was causing errors because the `members` list was empty.

**Comment by dibyom** [comment]
> {"component":"unset","file":"prow/cmd/peribolos/main.go:202","func":"main.main","level":"fatal","msg":"Configuration failed: failed to configure tektoncd teams: failed to update resolution.maintainers members: failed to list 0(resolution.maintainers) members: return code not 2XX: 404 Not Found","severity":"fatal","time":"2022-01-27T13:34:33Z"}

**Comment by afrittoli** [comment]
I'm not sure how to proceed on this TBH -  I will file an issue with peribolos, but until then how do we unlock this?
Perhaps we could use tide [`context_options`](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/config.md) to mark the job as not required for now? @vdemeester wdyt?

**Comment by vdemeester** [comment]
> I'm not sure how to proceed on this TBH - I will file an issue with peribolos, but until then how do we unlock this? Perhaps we could use tide [`context_options`](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/config.md) to mark the job as not required for now? @vdemeester wdyt?

Yeah I think we should force the PR to merge somehow indeed 👼🏼

**Comment by vdemeester** [comment]
(Actually might not work as it's not managed by prow..)

**Comment by afrittoli** [comment]
Uhm, I can override it by triggering a "passed" status updated from the cluster... it will solve the problem for this PR but not the general one. 
I'll do that now and create an issue to track the problem.

**Comment by afrittoli** [comment]
```
tkn task start github-set-status --use-taskrun 4ef35280-84e1-pull-community-org-validation-failure -n tekton-ci -p STATE=success
```

**Comment by afrittoli** [comment]
FYI https://github.com/tektoncd/community/issues/622

**Comment by vdemeester** [comment]
> Uhm, I can override it by triggering a "passed" status updated from the cluster... it will solve the problem for this PR but not the general one. I'll do that now and create an issue to track the problem.

we need to "automate" this in some dark corner of slack or somewhere 😬

**Comment by sbwsg** [comment]
Thanks for moving this through @afrittoli @vdemeester !

**Pull request merged**
